src/pages/workspaces/workspace/workflows/LaunchAnalysisModal.js

```diff
@@ -88,10 +108,20 @@ export default ajaxCaller(class LaunchAnalysisModal extends Component {
         this.createSetAndLaunch(entities)
       }
     } else if (type === EntitySelectionType.processFromSet) {
-      const { entityType, name } = selectedEntities
-      this.launch(entityType, name, `this.${rootEntityType}s`)
+      if (_.size(selectedEntities) === 1) {
+        const { entityType, name } = selectedEntities[0]
+        this.launch(entityType, name, `this.${rootEntityType}s`)
+      } else {
+        const entities = _.reduce((acc, { attributes: { [`${rootEntityType}s`]: { items } } }) => _.concat(acc, _.map('entityName', items)), [],
```

Comment:
TOL: I finally figured out what this is doing. Can it be done with `_.flatMap`? Or can it `_.flow` somehow?